coalib/parsing/FilterHelper.py

```diff
@@ -14,7 +15,21 @@ def is_valid_filter(filter):
     return filter in available_filters
 
 
-def apply_filter(filter_name, filter_args, all_bears=None):
+def apply_filter(filter_name, filter_args, all_bears=None, filter_function):
+    def filter_section_bears(bears, args):
+        return {section:
+                tuple(bear for bear in bears[section]
+                      if filter_function(bear, args))
+                for section in bears}
+
+    @wraps(filter_function)
+    def filter_wrapper(section_to_bears_dict, args):
```

Comment:
We don't this wrapper anymore. It was used to create annotation. Please inline this as well.